src/styles/main.scss

```
@@ -403,7 +402,9 @@ div.table-filter {
     }
 
     .ng-multi-value {
-      align-self: center;
+      display: flex;
```

I would also suggest adding a slight `gap` here:
```suggestion
      display: flex;
      gap: 4px;
```